environments/hca-prod/terraform/dagster.tf

```diff
@@ -58,3 +58,13 @@ resource google_storage_bucket_iam_member hca_dagster_staging_bucket_iam {
   role   = "roles/storage.admin"
   member = "serviceAccount:${module.hca_dagster_runner_account.email}"
 }
+
+resource google_storage_bucket_iam_member hca_dagster_temp_bucket_iam {
+  provider = google-beta.target
```

Comment:
Can you change this to the default google provider?

Reply:
i can, but it'll have to be a mass update since there's no default google provider currently specified for the project, and i don't know whether there'll be any feature drift as a result.

Reply:
cool, let's tackle that as part of the already ticketed work to do so.